psse/psse-model/src/main/java/com/powsybl/psse/model/PsseRawReader.java

```diff
@@ -85,122 +321,108 @@ public void handleError(DataProcessingException error, Object[] inputRow, Parsin
         return records;
     }
 
-    private PsseCaseIdentification readCaseIdentification(BufferedReader reader) throws IOException {
-        String line = readLineAndRemoveComment(reader);
-        PsseCaseIdentification caseIdentification = parseRecord(line, PsseCaseIdentification.class);
-        caseIdentification.setTitle1(reader.readLine());
-        caseIdentification.setTitle2(reader.readLine());
-        return caseIdentification;
+    private static String removeComment(String line) {
+        int slashIndex = line.lastIndexOf('/');
+        if (slashIndex == -1) {
+            return line;
+        }
+        return line.substring(0, slashIndex);
```

Comment:
Are you really sure this is correct: I wonder if the `lastIndexOf` is correct, or if you should find the first occurence of `/` ? What if I have a comment that contain a slash?

Reply:
Done. Change to the  first occurrence.